### ISSUE TITLE
Authentication Issue

### ISSUE THREAD

**Issue by rjkunde** [mozilla/openbadges-backpack#1138]
User is able to login to the Mozilla backpack website, but not in the modal window popup when accepting a badge.

This is what is displayed:
![image](https://cloud.githubusercontent.com/assets/10871915/26469315/2ddd79f6-415f-11e7-9e39-1d023af47571.png)

**Comment by sanderaido** [comment]
Scroll down...?

**Comment by rjkunde** [comment]
![fail2](https://user-images.githubusercontent.com/10871915/26988808-612441c4-4d16-11e7-9226-7e0776c6f9e6.PNG)

When scrolling down, and clicking the Login button, here is the error received.

I found out it wasn't due to the user not knowing their account/password.

![success firefox](https://user-images.githubusercontent.com/10871915/26988941-d6879182-4d16-11e7-98df-ca4f4aae5874.PNG)

Oddly enough, we tried using Firefox, instead of chrome, and the modal login button was working.  I went as far as to remove chrome completely from her Mac, and reinstall it.  This didn't solve the problem.

Then I had her use my machine (trying chrome in both windows, and OSX), and she was able to login via the modal window without issue.

This problem seems isolated to her operating system/browser configuration.

I'll go ahead and close this ticket.
